shared/actions/__tests__/team-building.test.tsx

```diff
@@ -95,28 +101,52 @@ const mockUserSearchRpcPromiseRpcPromise = (
   }
 }
 
+const expectedGithub: Array<Types.User> = [
+  {
+    id: 'marcopolo@github+marcopolo',
```

Comment:
@buoyad ^ here.

this file also got some more typing because there were issues, not affecting testing though